wire-compiler/api/wire-compiler.api

```diff
@@ -207,11 +208,12 @@ public final class com/squareup/wire/schema/WireRun {
 	public final fun component7 ()Ljava/lang/String;
 	public final fun component8 ()Ljava/lang/String;
 	public final fun component9 ()Ljava/util/List;
-	public final fun copy (Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/util/Map;Z)Lcom/squareup/wire/schema/WireRun;
-	public static synthetic fun copy$default (Lcom/squareup/wire/schema/WireRun;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/util/Map;ZILjava/lang/Object;)Lcom/squareup/wire/schema/WireRun;
+	public final fun copy (Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/util/Map;ZLjava/util/List;)Lcom/squareup/wire/schema/WireRun;
```

Comment:
Why is `WireRun` a data class anyway???

Reply:
Get rid of it! No `data class`es in public APIs!